### PR TITLE
Add script that extracts xml field formats.

### PR DESCRIPTION
**Pull request by Johannes-Sahlmann**

### CODE REVIEW

File: examples/extract_xml_field_formats.py [hunk 1]
```diff
@@ -123,7 +117,15 @@ def read_xml_field_formats(instrument, filename, verbose=False):
 
 
 def show_xml_field_formats(xml_formats, reference_instrument_name='NIRISS', out_dir=None):
+    """Print field formats to screen and write to file.
+
+    Parameters
```
Comment (shanosborne) [inline examples/extract_xml_field_formats.py]: It's a good practice to include the type (str, float, etc) and description for each of the parameters if you’re going to list them

File: examples/extract_xml_field_formats.py [hunk 2]
```diff
@@ -155,44 +157,12 @@ def show_xml_field_formats(xml_formats, reference_instrument_name='NIRISS', out_
                                            format='ascii.basic', delimiter=',', overwrite=True)
 
 
-xml_formats = {}
-for instrument in ['NIRISS', 'FGS', 'NIRSpec']:
-
-    siaf = pysiaf.Siaf(instrument)
+xml_formats = OrderedDict()
```
Comment (shanosborne) [inline examples/extract_xml_field_formats.py]: I’m not sure exactly how to want to run this file, but this line through the end of the file may be good to put under `if __name__ == "__main__”:`

File: examples/extract_xml_field_formats.py (hunk 1 above)
Comment (Johannes-Sahlmann) [reply examples/extract_xml_field_formats.py]: Done.

File: examples/extract_xml_field_formats.py (hunk 2 above)
Comment (Johannes-Sahlmann) [reply examples/extract_xml_field_formats.py]: Right. Thanks.

**Comment by Johannes-Sahlmann** [comment]
Thank you @shanosborne for the review. I addressed your comments and will merge this as soon as the tests pass.

**Pull request merged**